qwik-graphql-tailwind/src/components/repo-header/index.tsx

```diff
@@ -11,17 +11,17 @@ interface RepoHeaderProps {
   stargazerCount: number | string;
   forkCount: number | string;
   watcherCount: number | string;
-  issuesCount: number;
-  prCount: number;
+  issuesCount: number | string;
+  prCount: number | string;
 }
 
 export const RepoHeader = component$(
   ({ name, owner, watcherCount, stargazerCount, forkCount, issuesCount, prCount }: RepoHeaderProps) => {
     const { pathname } = useLocation();
     const basePath = `${owner}/${name}`;
     const tabList = createTabList({
-      issueCount: issuesCount,
-      pullRequestCount: prCount,
+      issueCount: Number(issuesCount),
+      pullRequestCount: Number(prCount),
```

Comment:
why do we have to convert to number? worst case scenario is that it would be Zero(0)

Reply:
The issue and PR count will be used for calculations(addition) and they made be a string, which will result in unwanted concatenation.

Reply:
Note that, because we can't pass values around via context we made use of query params.